frontend/src/components/Note.jsx

```diff
@@ -71,8 +82,8 @@ export default function Note({ role, id, group, user, refetchGroup }) {
     try {
       await saveNote();
```

Comment:
Unsafe usage of `getUserIdToken` in `saveNote` and `loadNote` - see above comment